findbugs/src/gui/edu/umd/cs/findbugs/gui2/help/About.html

```diff
@@ -8,27 +8,27 @@
 
 
 <p align="center">
-						<a href="http://findbugs.sourceforge.net/"><img src="umdFindbugs.png"
-								alt="FindBugs logo" border="0" /> </a>
+						<a href="https://github.com/spotbugs/"><img src="umdFindbugs.png"
```

Comment:
we probably should replace `umdFindbugs.png` with the spotbugs logo (there are a bunch in the repository under different directories), with apropriate copies of [our-logo](https://github.com/spotbugs/spotbugs.github.io/blob/master/spotbugs-icon.png) in a proper size. Will add a task to do so during the week.